### PR TITLE
CORE-185: endpoint to get launch info for a submission

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/apps/persistence/submissions.clj [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+(defn get-submission-by-id [submission-id]
+  (some-> (select* :submissions)
+          (fields :submission)
+          (where {:id (uuidify submission-id)})
+          select
+          first
+          :submission
+          .getValue
+          (json/decode true)))
```
Comment (psarando) [inline src/apps/persistence/submissions.clj]: 👍

File: src/apps/service/apps/jobs/params.clj [hunk 2]
```diff
@@ -105,35 +108,56 @@
...
 
 (defn- update-app-props
-  [config props]
-  (map (partial update-prop config) props))
+  [user config props]
+  (map (partial update-prop user config) props))
 
 (defn- update-app-group
-  [config group]
-  (update-in group [:parameters] (partial update-app-props config)))
```
Comment (psarando) [inline src/apps/service/apps/jobs/params.clj]: 👍

File: src/apps/service/apps/de/job_view.clj [hunk 3]
```diff
@@ -36,21 +37,11 @@
...
-       (if (#{"ERR_NOT_READABLE" "ERR_DOES_NOT_EXIST"} error-code)
-         false
-         (throw+))))))
-
 (defn- get-default-value
   [user type values]
   (let [default-value (mp/get-default-value type values)]
     (if (util/input-type? type)
-      (when (and default-value (path-accessible? user default-value))
+      (when (and default-value (paths-accessible? user default-value))
```
Comment (psarando) [inline src/apps/service/apps/de/job_view.clj]: 👍

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**